public/app/features/alerting/NotificationsListPage.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, FC, useEffect } from 'react';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import Page from 'app/core/components/Page/Page';
+import { getBackendSrv } from '@grafana/runtime';
+import { useAsyncFn } from 'react-use';
+import { useNavModel } from 'app/core/hooks/useNavModel';
+import { HorizontalGroup, Button } from '@grafana/ui';
+import { AlertNotification } from 'app/types/alerting';
+
+const deleteNotification = async (id: number) => {
+  return await getBackendSrv().delete(`/api/alert-notifications/${id}`);
+};
+
+const getNotifications = async () => {
+  return await getBackendSrv().get(`/api/alert-notifications`);
+};
+
+const NotificationsListPage: FC = () => {
+  const navModel = useNavModel('channels');
+
+  const [notifications, setNotifications] = useState<AlertNotification[]>();
+  const [state, fetchNotifications] = useAsyncFn(getNotifications);
+  useEffect(() => {
+    fetchNotifications().then(res => {
+      setNotifications(res);
+    });
+  }, []);
+
+  console.log(notifications);
+  return (
+    <Page navModel={navModel}>
+      <Page.Contents>
+        {state.error && <p>{state.error}</p>}
+        {!!notifications?.length && (
```

Comment:
This doesn't feel correct, but it kept rendering when the array was empty

Reply:
If you try to initialize  `notifications` to an empty array, then I think you could do:
```suggestion
        {notifications.length && (
```